src/integration-test/helpers/saucelabsReporter.js

```diff
@@ -12,7 +12,7 @@ function reportBuildResultToSaucelabs (result) {
   const sauceUsername = process.env.SAUCELABS_USERNAME
 
   request.put({
-    uri: `https://eu-central-1.saucelabs.com/rest/v1/${sauceUsername}/jobs/${sessionId}`,
+    uri: `https://eu-central-1.saucelabs.com/rest/v1/'+${sauceUsername}+'/jobs/'+${sessionId}`,
```

Comment:
i dont think this is the issue - that command is fine. the issue is here:

cmc-citizen-frontend/Jenkinsfile_nightly

Line 48 in 6b81480

 env.SAUCELABS_USERNAME = 'username' 
if these environment variables are automatically added by jenkins then just delete from our nightly jenkinsfile.

Reply:
Thanks for the updated had deleted this two-line 48 and 49